<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticDialogViewClass</class>
 <widget class="QDialog" name="StatisticDialogViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StatisticDialogView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Наименование:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Вид манипулация:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Филтър по възраст:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="procedureCombo">
           <item>
            <property name="text">
             <string>Обща</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Обща (на зъб)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Обтурация</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Екстракция</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ендодонтско лечение</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Корона</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Имплант</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Мост</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Фибромост</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="nhifCheckBox">
           <property name="text">
            <string>Преброй само манипулации по НЗОК</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QComboBox" name="ageCombo">
           <item>
            <property name="text">
             <string>Всички</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Непълнолетни</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Пълнолетни</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Филтър по зъби:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="toothCombo">
           <item>
            <property name="text">
             <string>Всички</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Постоянни</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Временни</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="nameGroup">
       <property name="title">
        <string>Филтър по манипулация</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListWidget" name="nameFilterList"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="diagnosisGroup">
       <property name="title">
        <string>Филтър по диагноза</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="diagnosisFilterList"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
